addons/zeus/CfgVehicles.hpp

```diff
@@ -317,4 +317,35 @@ class CfgVehicles {
     class ModuleArsenal_F: Module_F {
         function=QFUNC(bi_moduleArsenal);
     };
+
+    class Man;
+    class CAManBase: Man {
+        class ACE_SelfActions {
+            class GVAR(create) {
+                displayName = CSTRING(CreateZeus);
+                condition = QUOTE(\
+                isNil QQGVAR(zeus) && {\
+                    switch (GVAR(canCreateZeus)) do {\
+                        case CAN_CREATE_ADMIN:   {isServer || {IS_ADMIN_LOGGED}};\
```

Comment:
`IS_ADMIN_LOGGED` is not config compatible as it has `"#shutdown"` inside it. Fix incoming

Reply:
Fixed in https://github.com/CBATeam/CBA_A3/pull/1048/files
Will apply that in my PR